crates/kernel/testsuite/moot/example.moot

```diff
@@ -1,14 +1,29 @@
-# This is a comment.
+// This is a comment.
 ; return 42;
 42
 
-# Empty lines are ignored
+// Empty lines are ignored
 
-# Both thrown and return exceptions can be matched with a simple error value
+// Both thrown and return exceptions can be matched with a simple error value
 ; eval();
 E_ARGS
 
-# Multi-line commands: continuation with `>`.
+// Multi-line commands: continuation with `>`.
 ; return 1 + 2 +
 > 3;
-6
\ No newline at end of file
+6
+
+// You can change the player used to execute commands
+@programmer
+; return player;
+#4
+; return {player.programmer, player.wizard, player.name};
+{1, 0, "programmer"}
+
+@nonprogrammer
```

Comment:
so this is a bit of a weird thing because what's getting tested here is the permissions of a non-programmer player to call bf_eval, not the actual thing being eval'd?

Reply:
Correct! It's currently the only thing you *can* do with `@nonprogrammer`, and maybe the only thing we can ever do in `kernel`.